settings.json

```diff
@@ -0,0 +1,122 @@
+[  
```

Comment:
Should this file live in our deployment configurations?

Reply:
Left as an example(and for CI will open additional PR to deployments configs